scripts/build-tarballs.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const shelljs = require('shelljs');
+const PackageUtilities = require('@lerna/project');
+
+const rootPath = path.join(__dirname, '..');
+const tarballsDistPath = path.join(rootPath, 'dist-tarballs');
+
+function extractTarball(packageInfo) {
+  const appPackageJsonPath = path.join(packageInfo.location, 'package.json');
+  const appPackageJson = require(appPackageJsonPath);
+  if (appPackageJson.private) return;
+
+  shelljs.exec('npm pack', { cwd: packageInfo.location, silent: true });
```

Comment:
Confused when we use execa and when shelljs now. Can you elaborate?

Reply:
I still prefer `shelljs`, because it has good cross OS support and it can run synchronously. For example commands like `shelljs.cp`, `shelljs.mv`, etc.

The `execa` can be used with Promises, which was good for the Listr implementation.

In general for scripts I would personally still prefer to use shelljs.